pkg/registry/core/datapartition/storage/storage.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2020 Authors of Arktos.
```

Comment:
# Copyright 2015 The Kubernetes Authors.
# Copyright 2020 Authors of Arktos - file modified.